### PR TITLE
Jan Newsletter - post/send Jan 24

### PR DESCRIPTION
**Pull request by jules32**
Hi @stefaniebutland , I've reworked the newsletter, thinking we can do another in March with Reflections dates. If this fits with your schedule this week, would you please:

- review and see any big things missing before we meet on Tuesday so we can discuss
- post/send Wednesday this week

Thank you!

### CODE REVIEW

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+We also have a [February Community Call](TODO LINK): **GitHub for NASA Openscapes community project management & calendaring - a work in progress**. We invite you to come see how the NASA Openscapes Mentors are co-organizing an open calendar and shared tasks — bring your questions and experience!
+
+We'll be sharing more events and goals for 2024 in the coming months.
+
+
+## Some 2023 Highlights
+
+We are continually iterating and documenting what works as we learn from experiences collaborating and teaching. This year we've focused on communicating this in a few talks, particularly [the Flywheel as an open source tool](https://openscapes.org/blog/2024-01-18-agu-talk-2023/) as we described in a talk at the AGU meeting and in [documenting things, particularly onboarding](https://openscapes.org/blog/2023-09-27-documenting-things-posit-conf/) at the Posit conference in September.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:29]:
```suggestion
We are continually iterating and documenting what works as we learn from our experiences collaborating and teaching. This year we've focused on communicating this in a few talks, emphasizing [the Flywheel as an open source tool](https://openscapes.org/blog/2024-01-18-agu-talk-2023/) as we described in a talk at the AGU fall meeting and [documenting things openly for future us](https://openscapes.org/blog/2023-09-27-documenting-things-posit-conf/), like onboarding, at the Posit conference in September.
```

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [ Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:31]:
```suggestion
[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review. NOAA Fisheries Mentors Community (https://openscapes.org/blog/2024-01-24-nmfs-mentors/)
```

Comment (stefaniebutland) [reply blog/2024-01-24-news-jan-2024/index.qmd:31]: add phrase about NMFS Mentors Community, link to post published same day

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+We launched **two new annual programs** in 2023: The [Pathway to Open Science](https://openscapes.org/blog/2023-04-26-pathways-report/) program for Black marine and environmental scientists that kicks off this week, and the [Reflections](https://openscapes.org/blog/2023-09-06-reflections-2023/) program that is a lightweight commitment for individuals to identify and plan around their workflow needs. We'll share registration information ahead of May 2024!
+[[[[- We are also planning to lead an open call for the Champions Program - with no bounds on affiliation or focus. We are looking for sponsors ]]]] do we include this?
+  
+We also invested in sustainability for the community. We worked with SGCI [Focus Week]() in San Diego in May and with Nancy Maron First Five Years in Fall. This work informed our [Challenge.gov submission](LINK TO GOOGLE DOC).  we built a new [Zenodo Openscapes Community](https://openscapes.org/blog/2023-05-31-zenodo-community/) to help find and cite Openscapes things, we [migrated our website to Quarto](TODO LINK TO BLOG),
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:38]:
```suggestion
We also invested in sustainability for the community. We worked with SGCI [Focus Week] (LINK) in San Diego in May and with Nancy Maron First Five Years in Fall. This work informed our [Challenge.gov submission] (LINK TO GOOGLE DOC). We set up a new [Zenodo Openscapes Community](https://openscapes.org/blog/2023-05-31-zenodo-community/) to help people find and cite Openscapes things and we [migrated our website from blogdown to Quarto](https://openscapes.org/blog/2023-08-03-openscapes-website/). 
```

Comment (stefaniebutland) [reply blog/2024-01-24-news-jan-2024/index.qmd:38]: add phrase on what challenge.gov is

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+[NASA Openscapes](https://nasa-openscapes.github.io) progress focused on learning and codeveloping with the broader open science community via hackdays focused on R, MATLAB, Cookbook, and 2i2c access policies. Mentors across 11 NASA data centers continue improving how we teach and support researchers in the Cloud via workshops, hackathons, and Champions Cohorts. California Water Boards supported [their second annual Champions cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! 
+
+We launched **two new annual programs** in 2023: The [Pathway to Open Science](https://openscapes.org/blog/2023-04-26-pathways-report/) program for Black marine and environmental scientists that kicks off this week, and the [Reflections](https://openscapes.org/blog/2023-09-06-reflections-2023/) program that is a lightweight commitment for individuals to identify and plan around their workflow needs. We'll share registration information ahead of May 2024!
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:35]:
```suggestion
We launched **two new annual programs** in 2023: the [Pathways to Open Science](https://openscapes.org/blog/2023-04-26-pathways-report/) program for Black marine and environmental scientists, with our second offering kicking off this week, and the [Reflections](https://openscapes.org/blog/2023-09-06-reflections-2023/) program that is a lightweight commitment for individuals to identify and plan around their workflow needs. We'll share registration information ahead of the next offering in May 2024!
```

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [ Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review.
+
+[NASA Openscapes](https://nasa-openscapes.github.io) progress focused on learning and codeveloping with the broader open science community via hackdays focused on R, MATLAB, Cookbook, and 2i2c access policies. Mentors across 11 NASA data centers continue improving how we teach and support researchers in the Cloud via workshops, hackathons, and Champions Cohorts. California Water Boards supported [their second annual Champions cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting!
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:33]:
```suggestion
[NASA Openscapes](https://nasa-openscapes.github.io) progress focused on learning and codeveloping with the broader open science community via hackdays focused on R, MATLAB, the [Earthdata Cloud Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/), and 2i2c Openscapes JupyterHub access policies. Mentors across 11 NASA data centers continue improving how we teach and support researchers in the Cloud via workshops, hackathons, and [Champions Cohorts](https://openscapes.org/blog/2023-08-01-nasa-champions/). It was truly energizing to come together in person with so many NASA Openscapes Mentors and collaborators at the summer ESIP and winter [AGU meetings](https://openscapes.org/blog/2023-12-10-agu-fall-meeting/)!

California Water Boards ran [their second annual Champions Cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! 
```

Comment (stefaniebutland) [reply blog/2024-01-24-news-jan-2024/index.qmd:33]: add Carpentries Instructor training. How many certified?

and [Engaging with a Cross-NASA Subcommunity
Recap of a Carpentries Community Session with NASA and Openscapes](https://openscapes.org/blog/2023-12-07-engaging-with-a-cross-nasa-subcommunity/)

Comment (stefaniebutland) [reply blog/2024-01-24-news-jan-2024/index.qmd:33]: need cohesion for paragraph starting "California Water Boards...". Maybe it belongs above with "[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, ..." that refers to cohorts and mentors activities.

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+[[[[- We are also planning to lead an open call for the Champions Program - with no bounds on affiliation or focus. We are looking for sponsors ]]]] do we include this?
+  
+We also invested in sustainability for the community. We worked with SGCI [Focus Week]() in San Diego in May and with Nancy Maron First Five Years in Fall. This work informed our [Challenge.gov submission](LINK TO GOOGLE DOC).  we built a new [Zenodo Openscapes Community](https://openscapes.org/blog/2023-05-31-zenodo-community/) to help find and cite Openscapes things, we [migrated our website to Quarto](TODO LINK TO BLOG), 
+
+We've been grateful to (re)connect with colleagues virtually and in person at ESIP, Carpentries, Posit, Shannon D, CSCCE Tool Tips, We moved to Mastodon and have found it challenging to reestablish the nexus of open source, environmental/Earth science, and inclusive learning that we had previously. In 2024 we will work to reconnect.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:40]:
```suggestion
We've been grateful to (re)connect with colleagues virtually and in person at ESIP, AGU, The Carpentries, Posit, Shannon D, and CSCCE Tools Trials. We moved our social media to Mastodon and have found it challenging to reestablish the nexus of open source, environmental/Earth science, and inclusive learning that we had previously. In 2024 we will work to reconnect.
```

**Review by stefaniebutland — COMMENTED** [review]
added / edited in suggested commits and a small number of comments.

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+We also have a [February Community Call] (TODO LINK): **GitHub for NASA Openscapes community project management & calendaring - a work in progress**. We invite you to come see how the NASA Openscapes Mentors are co-organizing an open calendar and shared tasks — bring your questions and your experience!
```
Comment (jules32) [inline blog/2024-01-24-news-jan-2024/index.qmd:22]: TODO Link!

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+We'll be sharing more events and goals for 2024 in the coming months.
+
+
+## Some 2023 Highlights
+
+We are continually iterating and documenting what works as we learn from our experiences collaborating and teaching. This year we've focused on communicating this in a few talks, emphasizing [the Flywheel as an open source tool](https://openscapes.org/blog/2024-01-18-agu-talk-2023/) as we described in a talk at the AGU fall meeting and [documenting things openly for future us](https://openscapes.org/blog/2023-09-27-documenting-things-posit-conf/), like onboarding, at the Posit conference in September.
+
+[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [ Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review.
```
Comment (jules32) [inline blog/2024-01-24-news-jan-2024/index.qmd:31]:
```suggestion
[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [ Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review. California Water Boards ran [their second annual Champions Cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! 
```

**Comment by jules32** [comment]
Hi @stefaniebutland ! On your final readthrough could you find a place to add https://www.annualreviews.org/doi/full/10.1146/annurev-marine-041723-094741?

**Review by stefaniebutland — COMMENTED** [review]
moar edits. almost done

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 9]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Openscapes Newsletter #8: Winter 2024"
+author:
+  - name: Julie Lowndes
+    orcid: 0000-0003-1682-3872
+  - name: Stefanie Butland
+    orcid: 0000-0002-5427-8951  
+date: 2024-01-24
+draft: true
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:9]:
```suggestion
```

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  
+We also invested in sustainability for the community. We worked with SGCI [Focus Week]() in San Diego in May and with Nancy Maron First Five Years in Fall. This work informed our [Challenge.gov submission](LINK TO GOOGLE DOC).  we built a new [Zenodo Openscapes Community](https://openscapes.org/blog/2023-05-31-zenodo-community/) to help find and cite Openscapes things, we [migrated our website to Quarto](TODO LINK TO BLOG), 
+
+We've been grateful to (re)connect with colleagues virtually and in person at ESIP, AGU, The Carpentries, Posit, Shannon D, and CSCCE Tools Trials. We moved our social media to Mastodon and have found it challenging to reestablish the nexus of open source, environmental/Earth science, and inclusive learning that we had previously. In 2024 we will work to reconnect.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:39]:
```suggestion
We've been grateful to (re)connect with colleagues virtually and in person at ESIP, AGU, The Carpentries, Posit, The Open Environmental Data Project, and CSCCE Tools Trials. We moved our social media to Mastodon and have found it challenging to reestablish the nexus of open source, environmental/Earth science, and inclusive learning that we had previously. In 2024 we will work to reconnect.
```

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and Fred Hutch, we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review. California Water Boards ran [their second annual Champions Cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! With NOAA Fisheries staff, we co-designed a [NOAA Fisheries Openscapes Mentors](https://openscapes.org/blog/2024-01-24-nmfs-mentors/) program  modeled from the [NASA Openscapes](https://nasa-openscapes.github.io/) Mentors Framework.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:31]:
```suggestion
[Openscapes Mentors](https://openscapes.org/team) were an important part of the Year of Open Science. Together, across NASA Earthdata, NOAA Fisheries, California Water Boards, Pathways to Open Science, EPA, and [Fred Hutch](https://openscapes.org/blog/2023-12-04-fred-hutch/), we learned [coaching skills that have made us better open data science mentors](https://openscapes.org/blog/2023-05-17-mentor-coach/), and co-wrote a preprint [Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) that is in review. California Water Boards ran [their second annual Champions Cohort](https://openscapes.org/blog/2023-12-13-waterboards/) to support their colleagues to meet open, data, and equity goals. NOAA Fisheries shared their Open Data and Open Science vision at the summer ESIP meeting! With NOAA Fisheries staff, we co-designed a [NOAA Fisheries Openscapes Mentors](https://openscapes.org/blog/2024-01-24-nmfs-mentors/) program  modeled from the [NASA Openscapes](https://nasa-openscapes.github.io/) Mentors Framework.
```

**Review by stefaniebutland — COMMENTED** [review]
adding links

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+
+We also have a [February Community Call](https://openscapes.org/events/2024-02-20-github-calendaring/): **GitHub for NASA Openscapes community calendaring & project management - An open work in progress**. We invite you to come see how the NASA Openscapes Mentors are co-organizing an open calendar and shared tasks — bring your questions and your experience!
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:21]:
```suggestion
Join us for our [February Community Call](https://openscapes.org/events/2024-02-20-github-calendaring/): **GitHub for NASA Openscapes community calendaring & project management - An open work in progress**. We invite you to come see how the NASA Openscapes Mentors are co-organizing an open calendar and shared tasks — bring your questions and your experience!

Seeing this paper "Welcoming More Participation in Open Data Science for the Oceans" ([Fredston and Lowndes, 2024](https://doi.org/10.1146/annurev-marine-041723-094741)) published in the Annual Review of Marine Science made a great start to the year!
```

File: blog/2024-01-24-news-jan-2024/index.qmd [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+
+We're kicking off 2024 with the second annual [**Pathways to Open Science**](https://openscapes.github.io/pathways-to-open-science/) program that starts this week! This is an ongoing collaboration with [Black in Marine Science (BIMS)](https://www.blackinmarinescience.org/), [Black Women in Ecology, Evolution, and Marine Science (BWEEMS)](https://www.bweems.org/), and HBCU marine science professors led by Ileana Fenwick, with Alex Davis, PhD and Aneese Williams. Thanks to new funding from the [Mozilla Foundation's Alumni Connection grants](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/), we're excited to collaborate with [PREreview](https://prereview.org/) and Antoinette Foster, PhD to bring expertise in values-guided career navigation and anti-racist scholarly review to the programming this year. [Register here](https://zoom.us/meeting/register/tJ0ldempqDIrHtQ4KqAbr442Tm0xl4-h_dtC) if you're a Black marine or environmental scientist and/or an HBCU student. If you are an ally, please help us amplify Black voices; we suggest allies do not sign up for this particular series.
```
Comment (stefaniebutland) [inline blog/2024-01-24-news-jan-2024/index.qmd:19]:
```suggestion
We're kicking off 2024 with the second annual [**Pathways to Open Science**](https://openscapes.github.io/pathways-to-open-science/) program that starts this week! This is an ongoing collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)), and HBCU marine science professors led by Ileana Fenwick, with Alex Davis, PhD and Aneese Williams. Thanks to new funding from the [Mozilla Foundation's Alumni Connection grants](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/), we're excited to collaborate with [PREreview](https://prereview.org/) and Antoinette Foster, PhD to bring expertise in values-guided career navigation and anti-racist scholarly review to the programming this year. [Register here](https://zoom.us/meeting/register/tJ0ldempqDIrHtQ4KqAbr442Tm0xl4-h_dtC) if you're a Black marine or environmental scientist and/or an HBCU student. If you are an ally, please help us amplify Black voices; we suggest allies do not sign up for this particular series.
```

**Pull request merged**